### PR TITLE
UI rewrite

### PR DESCRIPTION
**Pull request by benbacardi**

### CODE REVIEW

File: Elsewhen.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1758,11 +1742,12 @@
 				INFOPLIST_KEY_CFBundleDisplayName = Elsewhen;
 				INFOPLIST_KEY_LSApplicationCategoryType = "public.app-category.utilities";
 				INFOPLIST_KEY_NSHumanReadableCopyright = "";
+				INFOPLIST_KEY_UISupportedInterfaceOrientations_iPhone = "UIInterfaceOrientationPortrait UIInterfaceOrientationPortraitUpsideDown";
 				LD_RUNPATH_SEARCH_PATHS = (
 					"$(inherited)",
 					"@executable_path/../Frameworks",
 				);
-				MACOSX_DEPLOYMENT_TARGET = 11.3;
+				MACOSX_DEPLOYMENT_TARGET = 12.0;
```
Comment (alexcfaber) [inline Elsewhen.xcodeproj/project.pbxproj:1750]: Intended?

File: Elsewhen/ElsewhenApp.swift [hunk 2]
```diff
@@ -26,7 +26,8 @@ struct ElsewhenApp: App {
         WindowGroup {
             #if os(macOS)
             MacContentView()
-                .frame(minWidth: 700, minHeight: 400)
+                .frame(minWidth: 400, minHeight: 500)
```
Comment (alexcfaber) [inline Elsewhen/ElsewhenApp.swift:29]: I'm assuming you tested the minimum width and height is enough? :)

**Comment by alexcfaber** [comment]
Looks good to me!

A generic comment is that we should maybe look in to having some guidelines on how we deal with macOS or iOS exceptions. It seems to be all over the place now, especially if more platforms get added, but also for readability always go

iOS, then MacOS, then Default.

**Review by alexcfaber — APPROVED** [review]

File: Elsewhen/ElsewhenApp.swift (hunk 2 above)
Comment (benbacardi) [reply Elsewhen/ElsewhenApp.swift:29]: I did!

**Pull request merged**